gensim/models/word2vec.py

```diff
@@ -136,111 +137,180 @@
 
 try:
     from gensim.models.word2vec_inner import train_batch_sg, train_batch_cbow
-    from gensim.models.word2vec_inner import score_sentence_sg, score_sentence_cbow
+    from gensim.models.word2vec_inner import score_sentence_sg, \
+        score_sentence_cbow
     from gensim.models.word2vec_inner import FAST_VERSION, MAX_WORDS_IN_BATCH
 
 except ImportError:
     # failed... fall back to plain numpy (20-80x slower training than the above)
     FAST_VERSION = -1
     MAX_WORDS_IN_BATCH = 10000
 
+
     def train_batch_sg(model, sentences, alpha, work=None, compute_loss=False):
-        """
-        Update skip-gram model by training on a sequence of sentences.
+        """Update skip-gram model by training on a sequence of sentences.
         Each sentence is a list of string tokens, which are looked up in the model's
         vocab dictionary. Called internally from `Word2Vec.train()`.
         This is the non-optimized, Python version. If you have cython installed, gensim
         will use the optimized version from word2vec_inner instead.
+
+        Parameters
+        ----------
+        model : :class:`~gensim.models.word2vec.Word2Vec`
+            `Word2Vec` instance.
+        sentences : iterable of iterables
+            Iterable of the sentences directly from disk/network.
+        alpha : float
+            Learning rate.
+        work : :class:`numpy.ndarray`
```

Comment:
please use `numpy.ndarray` without `:class:` for some external links (this doesn't work as link anyway)